developer-docs-site/docs/nodes/indexer-fullnode.md

```diff
@@ -80,8 +84,26 @@ Docker is not yet supported for indexer fullnode. In the above step 2, use only
         emit_every: 500
     ```
 
-10. Run the indexer fullnode with:
+1. Run the indexer fullnode with:
     ```bash
-    cargo run --bin aptos-node --features "indexer"  -- --config </path/to/fullnode.yaml>`
+    cargo run --bin aptos-node --features "indexer"  -- --config /path/to/fullnode.yaml
```

Comment:
cargo run -p aptos-node --features "indexer" --release -- -f /path/to/fullnode.yaml